### PR TITLE
Linear search #2179

### LINKED ISSUE #2179: Linear Search
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [x] Dart - @keshakaneria
- [x] Go
- [x] Java
- [x] Julia
- [x] JS
- [x] Kotlin
- [x] Php
- [x] Python
- [x] Readme
- [x] Ruby
- [x] Rust - @raksha009
- [x] TypeScript - @mruna7

### PR DESCRIPTION
**Pull request by mruna7**
Fixes #2179

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- typescript
- 

#### Files Added:
- Linear_Search.ts
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by mruna7** [comment]
@asha15  mam @singh-shreya6 mam please review this!!

File: Linear_Search/Linear_Search.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    return -1;
+}  
+   
+       
+var result = 0  
+var n = 0	 
+var val = 5  //input value 
+console.log(" value to be found: ", val);
+var n: number = 10 //no. of elements in the array
+var arr = [1, 2, 3, 4, 5, 6,7,8,9,10]; //input array
```
Comment (atarax665) [inline Linear_Search/Linear_Search.ts:19]: Give space after each comma.

**Comment by mruna7** [comment]
> Make changes and squash your commits.

yes @atarax665  sir sure . Please re-review it

File: Linear_Search/Linear_Search.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/*    Implenetation of LinearSearch algorithm to find value 5 in a given array of numbers    */
+
+function LinearSearch( arr:number[] , n:number ,  val:number ) :number
```
Comment (asha15) [inline Linear_Search/Linear_Search.ts:3]: please remove the space before comma 
remove the space around parenthesis

Comment (mruna7) [reply Linear_Search/Linear_Search.ts]: is it okay now @asha15

File: Linear_Search/Linear_Search.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/*    Implenetation of LinearSearch algorithm to find value 5 in a given array of numbers    */
+
+function LinearSearch(arr:number[], n:number,  val:number) :number
+{     
+    for (let i = 0; i < n; i++) 
+    {
+        if(val == arr[i])
```
Comment (singh-shreya6) [inline Linear_Search/Linear_Search.ts:7]: Please give one space after if like if (val == arr[i])

File: Linear_Search/Linear_Search.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
+/*    Implenetation of LinearSearch algorithm to find value 5 in a given array of numbers    */
```
Comment (singh-shreya6) [inline Linear_Search/Linear_Search.ts:1]: Typo in word Implementation

File: Linear_Search/Linear_Search.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+       
+var result = 0  
+var n = 0	 
+var val = 5  //input value 
+console.log(" value to be found: ", val);
+var n: number = 10 //no. of elements in the array
+var arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]; //input array
+console.log(" Array of numbers is: ", arr);
+result = LinearSearch(arr, n, val)  
+if (result != -1)
```
Comment (singh-shreya6) [inline Linear_Search/Linear_Search.ts:22]: Please convert this to :
if (result != -1) {
    console.log(" value ", val," found at index no. ", result," in the array ");
} else {
    console.log(" value ", val," not found ");
}

Comment (singh-shreya6) [reply Linear_Search/Linear_Search.ts:22]: keep the indentation in console.log line.

Comment (mruna7) [reply Linear_Search/Linear_Search.ts:22]: @singh-shreya6  mam is it okay now?

Comment (mruna7) [reply Linear_Search/Linear_Search.ts:22]: > keep the indentation in console.log line.

@singh-shreya6  mam can I squash the commits now?

**Comment by mruna7** [comment]
> yes, please squash your commits.

yes @singh-shreya6 mam done!!

**Pull request merged**